src/odo.ts

```diff
@@ -92,10 +92,10 @@ export const Command = {
         `oc get service ${component}-${app} --namespace ${project} -o jsonpath="{range .spec.ports[*]}{.port}{','}{end}"`,
     linkComponentTo: (project: string, app: string, component: string, componentToLink: string, port?: string) =>
         `odo project set ${project} && odo application set ${app} && odo component set ${component} && odo link ${componentToLink} --wait${port ? ' --port ' + port : ''}`,
-    pushComponent: (project: string, app: string, component: string) =>
-        `odo push ${component} --app ${app} --project ${project}`,
-    pushLocalComponent: (project: string, app: string, component: string, location: string) =>
-        `${Command.pushComponent(project, app, component)} --local ${location}`,
+    pushComponent: (project: string, app: string, component: string, level: number) =>
+        `odo push ${component} --app ${app} --project ${project} -v ${level}`,
```

Comment:
rather than changing every factory method and adding level parameter, lets just get verbosity level directly from workspace settings here
``` typescript
`odo push ${component} --app ${app} --project ${project} -v ${Odo.VERBOCITY}`,
```

``` typescript
public get VERBOCITY() {
    return workspace.getConfiguration(........);
}
```

See docs for configuration API [here](https://code.visualstudio.com/api/references/vscode-api#workspace).